airflow/hooks/hive_hooks.py

```diff
@@ -487,22 +487,22 @@ def get_results(self, hql, schema='default', arraysize=1000):
                 'data': [],
                 'header': [],
             }
+            cur = conn.cursor()
```

Comment:
I just looked at the dbapi and there are no close() method of instructions to run in a context manager. Therefor this LGTM.